src/tasks.py

```diff
@@ -559,6 +559,24 @@ def get_metrics(self, reset=False):
         return metrics
 
 
+class OpenAIEdgeProbingTask(EdgeProbingTask):
+    """Version of EdgeProbingTask that loads BPE-tokenized data."""
+    @property
+    def tokenizer_name(self):
+        return "OpenAI.BPE"
+
+
+# Register copies of all edge probing tasks.
```

Comment:
it's a little weird to me to have executed code this deep into a file that otherwise just looks like class definitions, but seems harmless enough

Reply:
Yeah, I don't love this either but it's just adding entries to the registry. I implemented it as a loop to avoid a (very) large block of boilerplate. Unfortunately, it does have to go at the end of the file since it requires the relevant classes to be defined.